src/plugins/ui_actions/public/cell_actions/components/cell_actions.tsx

```diff
@@ -110,9 +110,11 @@ export const CellActions: React.FC<CellActionsProps> = ({
     [field, triggerId, metadata]
   );
 
+  const dataTestSubj = `render-content-${field.name}`;
```

Comment:
could we use camel case here ([style guide](https://github.com/elastic/kibana/blob/main/STYLEGUIDE.mdx#camel-case-id-and-data-test-subj)), and maybe mention "cellActions" as well: `cellActions-renderContent-${field.name}`